coops-ui/src/views/AnnualReport.vue

```diff
@@ -206,15 +206,15 @@
           v-if="isAnnualReportEditable"
           :disabled="!isSaveButtonEnabled || busySaving"
           :loading="saving"
-          @click="onClickSave"
+          @click="onClickSave()"
```

Comment:
Ive noticed the added parenthesis throughout though i don't fully understand the difference in this context. Essentially if feels like the same thing with or without them, so i'm just curious? Without parentheses we're referencing the method and when we have them we're invoking the method itself, thats about it for my understanding of it :)

Reply:
Both ways seem to work but I prefer to include the parentheses as it makes it more explicit that we're calling a method.